README.md

```diff
@@ -1,2 +1,47 @@
-# intern-project-20
-Repository for the Group Project of the Intern Class of Summer 2020 
+# Eco Cart
+> Eco Cart is a data visualaztion app that allows users to measure the 
+> environmental impact of their groceries. Drawing from [this dataset](https://science.sciencemag.org/content/360/6392/987/tab-figures-data),
+> Eco Cart determines the environmental cost of certain grocery items based on their
+> water usage, eutrophication, land usage, and carbon emissions.
+
+
+## Dependencies
+
+* Node.js (12.18.0)
+
+## Install
+
+Pull repo down:
+
+```
+git clone https://github.com/vigetlabs/EcoCart.git
+cd EcoCart
+```
+
+Install packages:
+
+```
+npm install
+```
+
+## Development
+
+Starting Server:
+
+```
+npm start
+```
+
+And open up your favorite browser to [http://localhost:8000](http://localhost:3000)
```

Comment:
We need to think about browser compatibility. E.g. I'm not sure if the tools we're using will work on I.E.